assets/osx/Alacritty.app/Contents/Info.plist

```diff
@@ -34,5 +34,23 @@
   <string>Alacritty</string>
   <key>NSRequiresAquaSystemAppearance</key>
   <string>NO</string>
+    <key>NSAppleEventsUsageDescription</key>
+    <string>An application in alacritty wants to use AppleScript.</string>
+    <key>NSCalendarsUsageDescription</key>
+    <string>An application in alacritty wants to use Calendar data.</string>
```

Comment:
`alacritty` should be capitalized. Both lowercase and uppercase spellings exist, but I think uppercase is the better choice here.

Reply:
It doesn't prompt on installation or startup. It only prompts when an application tries to access those special privileges. I wrote a quick script to access the camera in the default Terminal.app (Since it was the only terminal emulator that I hadn't granted camera permission yet) to give you an idea of what it would look like.  

<img width="448" alt="screen shot 2018-11-08 at 19 44 49" src="https://user-images.githubusercontent.com/17736112/48236134-e2849100-e38e-11e8-82c0-8ecf17361990.png">

Reply:
Thanks for the feedback, then the wording should be fine. Alacritty should be capitalized though.

`Calendar data` should also be changed to `calendar data`.